[Flaky] rspec ./spec/system/admin/order_spec.rb:146

### What we should change and why (this is tech debt)
<!-- The use of X changed and part y is now redundant. We can remove it and... -->
```
Failures:

  1) 
    As an administrator
    I want to create and edit orders
 can add a product to an existing order
     Failure/Error: expect(page).to have_selector 'td', text: product.name
       expected to find visible css "td" with text "Product #4 - 3935" but there were no matches. Also found "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", "<<ERROR>>", which matched the selector but not all filters. 
     
     [Screenshot Image]: /home/runner/work/openfoodnetwork/openfoodnetwork/tmp/capybara/screenshots/failures_r_spec_example_groups_as_an_administrator_i_want_to_create_and_edit_orders_can_add_a_product_to_an_existing_order_285.png

     
     # ./spec/system/admin/order_spec.rb:155:in `block (2 levels) in <main>'
     # ./spec/system/support/cuprite_setup.rb:41:in `block (2 levels) in <top (required)>'

Finished in 4 minutes 42.8 seconds
52 examples, 1 failure

Failed examples:

rspec ./spec/system/admin/order_spec.rb:146 # 
    As an administrator
    I want to create and edit orders
 can add a product to an existing order
I, [2023-05-03T14:18:49.146894 #2963]  INFO -- : [knapsack_pro] To retry all the tests assigned to this CI node, please run the following command on your machine:
I, [2023-05-03T14:18:49.146956 #2963]  INFO -- : [knapsack_pro] bundle exec rspec --format progress --default-path spec "spec/system/admin/order_spec.rb"
Stopped processing SimpleCov as a previous error not related to SimpleCov has been detected
Coverage report generated for RSpec to /home/runner/work/openfoodnetwork/openfoodnetwork/coverage. 6888 / 24146 LOC (28.53%) covered.
Error: Process completed with exit code 1.
```

### Context
<!-- This came up in PR #3451#issuecomment-461616636. -->
https://github.com/openfoodfoundation/openfoodnetwork/actions/runs/4872903815/jobs/8691745034

### Impact and timeline
<!-- Example: This simplifies the upgrade to Spree 3 and should be done after upgrading to Spree 2. -->

--- a/spec/system/admin/order_spec.rb
+++ b/spec/system/admin/order_spec.rb
@@ -179,9 +179,9 @@ describe '
     visit spree.edit_admin_order_path(order)
 
     select2_select product.name, from: 'add_variant_id', search: true
-
     find('button.add_variant').click
     # Wait for JS
+    sleep(1)
     page.has_selector?("table.index tbody tr td")
     expect(page).to have_selector 'td', text: product.name
     expect(order.line_items.reload.map(&:product)).to include product

